src/marqo/_httprequests.py

```diff
@@ -38,7 +43,14 @@ def send_request(
         path: str,
         body: Optional[Union[Dict[str, Any], List[Dict[str, Any]], List[str], str]] = None,
         content_type: Optional[str] = None,
+        max_retry_attempts: Optional[int] = None,
+        max_retry_backoff_seconds: Optional[int] = None
     ) -> Any:
+        if max_retry_attempts is None:
```

Comment:
Can you add a unit test for this function that tests these defaults? Try not passing `max_retry_attempts` and `max_retry_backoff_seconds` and ensure that it defaults to the constants you set.